### ISSUE TITLE
Search does nothing?

### ISSUE THREAD

**Issue by hubertlepicki** [fly-apps/live_beats#73]
https://github.com/fly-apps/live_beats/assets/33274/268d2b04-4013-4890-9e45-1218920ea19b

I think it does nothing just confuses people?
https://github.com/fly-apps/live_beats/blob/63bbecf1b82d42c81420e28168abff14acdea125/lib/live_beats_web/components/layouts/live.html.heex#L109

**Comment by chrismccord** [comment]
Yeah this is a carry over from the tailwindui copy pasta. It does nothing indeed and could be removed. Please send a PR!
